### PR TITLE
feat(core): :sparkles: linear-gradient styles documentation added

### PR DESCRIPTION
**Pull request by Edithmark42**
linear-gradient styles documentation added

Ref #93

### CODE REVIEW

File: app/components/styles-docs/StyleUtilitiesSamples.js [hunk 1]
```diff
@@ -1039,6 +1061,166 @@ export const CLASS_SPECIFIC_SAMPLE_COMPONENT = {
       );
     },
   },
+  MARGIN__ENDS_WITH___N1: {
```
Comment (techoneel) [inline app/components/styles-docs/StyleUtilitiesSamples.js:1064]: unnecessary margin related code added here

**Review by techoneel — CHANGES_REQUESTED** [review]
chnages required because different issue's code came here

File: app/components/styles-docs/StyleUtilitiesSamples.js (hunk 1 above)
Comment (Edithmark42) [reply app/components/styles-docs/StyleUtilitiesSamples.js]: the unrelated code of margin was removed as it is not related to this particular issue.

**Review by techoneel — APPROVED** [review]
good to go

**Pull request merged**